### DISCUSSION TITLE
Google analytics plugin stopped working

### DISCUSSION THREAD

**Discussion by liliakai** [GitbookIO/community#765] [0. General bugs and questions]
My GA dashboard stopped receiving (almost all) data a few days ago, and I can no longer load the Google analytics plugin page for the site: https://app.gitbook.com/o/WQpOq5ZFue4N6m65QCJq/integrations/googleanalytics/configuration/site/site_ZWiTf

**Comment by liliakai** [comment]
Also, page view counts in the gitbook in-app analytics are drastically higher than what GA shows, by a factor of 2-5x. Is there any reason for this disparity?

**Comment by valentin0h** [comment]
> My GA dashboard stopped receiving (almost all) data a few days ago, and I can no longer load the Google analytics plugin page for the site: https://app.gitbook.com/o/WQpOq5ZFue4N6m65QCJq/integrations/googleanalytics/configuration/site/site_ZWiTf

Thanks for reporting @liliakai . We identified the issue and have applied a fix. It should be fixed on your side within the next few hours.

**Reply by liliakai** [reply]
Still not resolved for me, seeing the same behavior.

**Reply by valentin0h** [reply]
There was some caching involved that needed to be cleared. I double-checked that events are collected now. Getting to full data analytics might take a bit more time until users' caches are invalidated. 

![CleanShot 2024-10-23 at 09 56 07](https://github.com/user-attachments/assets/1658bcd8-1993-49ed-98e0-35b3c5fa717e)
